lib/utils.js

```diff
@@ -16,6 +17,7 @@ const CONFIG_DIR = '.losant';
 const configName = (file) => {
   return file || '.application.yml';
 };
+const DEFAULT_URL = process.env.TESTING_URL || 'https://api.losant.com';
```

Comment:
Because api.losant.com is already the default URL. api.losant.space is hard coded for all tests so it's the default testing url. Could be DEFAULT_TEST_URL or something.

Reply:
Why does it have to have the word test in there, I understand you're doing it because of a test, but could we not have someone who wants the API default to be something else? I would call it DEFAULT_API_URL.